### PR TITLE
containerize-your-workload-where-applicable.md

### PR DESCRIPTION
**Pull request by yelghali**
@dubrie

### CODE REVIEW

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
```
Comment (dubrie) [inline docs/catalog/cloud/containerize-your-workload-where-applicable.md:7]: please include `cloud` in the tag list as well

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Containerize your workload where applicable
```
Comment (dubrie) [inline docs/catalog/cloud/containerize-your-workload-where-applicable.md:13]: simplify to `Containerize your workloads` and move the applicability discussion to the solution and/or considerations and assumptions sections below.

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Description
+Containers may reduce unnecessary resource allocation and increase utilization of the compute resources, as they allow for bin packing and require less compute resources than virtual machines.
+
+## Solution
+Use [Draft](https://github.com/Azure/draft) to simplify application containerization by generating Dockerfiles and Kubernetes manifests.
```
Comment (dubrie) [inline docs/catalog/cloud/containerize-your-workload-where-applicable.md:19]: Vendor-specific implementations like Draft are not being accepted as patterns. Can this be presented in a vendor-agnostic way?

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, containerizing workloads will impact:
+
+- `E`: By reducing CPU and RAM utilization, we reduce the amount of energy required.
```
Comment (dubrie) [inline docs/catalog/cloud/containerize-your-workload-where-applicable.md:27]: I think you mean to say "CPU and RAM usage" instead of utilization.

**Review by dubrie — CHANGES_REQUESTED** [review]
@yelghali Changes requested from initial review

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md (hunk 3 above)
Comment (yelghali) [reply docs/catalog/cloud/containerize-your-workload-where-applicable.md:19]: Draft is actually OSS and works with any Kubernetes cluster (even though proposed by Microsoft). we could remove the link but i think it's a good enabler for customers

Comment (yelghali) [reply docs/catalog/cloud/containerize-your-workload-where-applicable.md:19]: and draft is meant as a proposal to help containrization (not the pattern per say)

**Comment by yelghali** [comment]
@dubrie,  thank you, done

**Comment by dubrie** [comment]
assigning to @dubrie for SME review

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- size:medium
+---
+
+# Containerize your workloads
+
+## Description
+Containers may reduce unnecessary resource allocation and increase utilization of the compute resources, as they allow for bin packing and require less compute resources than virtual machines.
+
+## Solution
+Use [Draft](https://github.com/Azure/draft) tool to simplify application containerization by generating Dockerfiles and Kubernetes manifests.
```
Comment (dubrie) [inline docs/catalog/cloud/containerize-your-workload-where-applicable.md:20]: Draft is a vendor-specific solution. Patterns can reference vendored tools as options or examples, but the pattern solution should be vendor-agnostic.

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md (hunk 3 above)
Comment (dubrie) [reply docs/catalog/cloud/containerize-your-workload-where-applicable.md]: Even as OSS, it is still vendor-specific as it is using a specific tool/app/service/product as the solution. A solution for a pattern should be "use this tool" but rather "use this approach". The solution here should be focusing on how you would containerize workloads. Draft can be used as an example of a tool that makes it simpler, but the solution section of the pattern needs to talk about the concepts (creating dockerfiles, manifests, etc)

File: docs/catalog/cloud/containerize-your-workload-where-applicable.md (hunk 5 above)
Comment (yelghali) [reply docs/catalog/cloud/containerize-your-workload-where-applicable.md]: proposal for phrasing:

Containerize your applications with their minimal dependencies. 

You can use [Draft](https://github.com/Azure/draft) tool to simplify application containerization by generating Dockerfiles and Kubernetes manifests.

**Comment by yelghali** [comment]
@dubrie added proposal for rephrasing pattern

**Comment by markus-ntt-seidl** [comment]
@yelghali Where did you add this proposal? I can't see it.

**Comment by franziska-warncke** [comment]
@markus-gsf-seidl Can you have a look?

**Review by markus-gsf-seidl — APPROVED** [review]
Maybe a sentence can be added/rewritten that containers itself do not reduce resource allocation, but allow resources to be used more flexible as workloads can be easily moved between machines.

Considerations could use a reference to k8s, for an advanced management of resources.

**Comment by franziska-warncke** [comment]
@markus-gsf-seidl Agree, I added this. One small thing, I did not added a reference to k8s itself, but mentioned a container orchastration system. How do you like it?

**Pull request merged**